builder/export_container

```diff
@@ -14,6 +14,8 @@ if [ -z "$1" ]; then
 else
     VERSION=$1
 fi
+CONTAINER_NAME=$(cat ${GIT_ROOT}/configuration/build_config.yaml | grep Base-Name: | awk -F " " '{print $2}')
```

Comment:
Should we use tooling like `yq` to read out the values from the YAML file?

Reply:
Hmm, your call.

It would be possible, see e.g. this commit & check https://github.com/antonjahn/ebcl_dev_container/commit/cd98ae548ece01420ba37c3cd5b295f32e336edc

| tool | Pro | Con |
|--------|--------|--------|
| awk | available on most linux boxes | syntax is not great |
| yq | can work with more complex yaml files, syntax is clean, is available on github runners | scripts are called locally and maintainers need to get yq as a dependency, okayish for modern ubuntu |